lib/modules/apostrophe-ui/views/components/tooltip.html

```diff
@@ -0,0 +1,3 @@
+{# this is cloned on the front-end and used to display tooltips for the apostrophe admin ui #}
+
+<div class="apos-hidden stuartttt apos-tooltip" data-apos-tooltip-template></div>
```

Comment:
We don't want to interfere with future web frameworks created in your honor.

Reply:
goo'bai